demisto_sdk/commands/common/errors.py

```diff
@@ -4394,3 +4406,29 @@ def pack_have_nonignorable_error(nonignorable_errors: List[str]):
     @error_code_decorator
     def correlation_rules_missing_search_window():
         return "The 'search_window' key must exist and cannot be empty when the 'execution_mode' is set to 'SCHEDULED'."
+
+    @staticmethod
+    @error_code_decorator
+    def command_reputation_output_capitalization_incorrect(
+        command_name: str,
+        invalid_outputs: list[str],
+        used_objects: set,
+    ):
+        return (
```

Comment:
I prefer the \ convention. using 3 times f notation looks like 3 separate return strings when skimming over the code.
something like this :
```
return (
            f"The {command_name} command returns the following reputation outputs:\n{invalid_outputs}\nwhich is\
             using mandatory objects:\n{used_objects} with incorrect capitalization. Please fix accordingly.\
             \nFor further information: {XSOAR_CONTEXT_AND_OUTPUTS_URL}"
        )
```
but i guess its a matter of style